cluster-scope/bundles/jaeger-operator/kustomization.yaml

```diff
@@ -3,5 +3,7 @@ kind: Kustomization
 namespace: openshift-distributed-tracing
 resources:
   - ../../base/core/namespaces/openshift-distributed-tracing
-  - ../../base/operators.coreos.com/subscriptions/jaeger-operator
-  - ../../base/operators.coreos.com/operatorgroups/jaeger-operator
+  - ../../base/operators.coreos.com/subscriptions/openshift-distributed-tracing-platform
+  - ../../base/operators.coreos.com/operatorgroups/openshift-distributed-tracing-platform
+  - ../../base/operators.coreos.com/subscriptions/openshift-distributed-tracing-data-collection
+  - ../../base/operators.coreos.com/operatorgroups/openshift-distributed-tracing-data-collection
```

Comment:
this file does not exist, and I don't think we need one for collection if we are installing it in the `openshift-operators` namespace

Reply:
fixed!